roadrunner/roadrunner.go

```diff
@@ -0,0 +1,102 @@
+package roadrunner
+
+import (
+	"fmt"
+
+	configImpl "github.com/roadrunner-server/config/v2"
+	endure "github.com/roadrunner-server/endure/pkg/container"
+	"github.com/roadrunner-server/errors"
+	"github.com/roadrunner-server/roadrunner/v2/internal/container"
+	"github.com/roadrunner-server/roadrunner/v2/internal/meta"
+)
+
+const (
+	rrPrefix string = "rr"
+)
+
+type RR struct {
+	container *endure.Endure
+	Version   string
+	BuildTime string
+}
+
+// NewRR creates a new RR instance that can then be started or stopped by the caller
+func NewRR(cfgFile string, override *[]string, pluginList []interface{}) (*RR, error) {
+	const op = errors.Op("new_rr")
```

Comment:
We may eliminate the RR-specific error package here since it'll be used by the apps that mostly use standard Go errors. So, fewer deps -> smaller package :)